.github/workflows/CD.yml

```diff
@@ -0,0 +1,43 @@
+name: CD
+
+on:
+  push:
+    branches:
+      - master
+      - feature/*
+  # Allows manual trigger of pipeline
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    name: Deploy to AKS
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2 
+
+      - name: Azure login
+        id: login
+        uses: azure/login@v1.4.3
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+      - name: Set AKS context
+        id: set-context
+        uses: azure/aks-set-context@v3
+        with:
+          resource-group: 'cdp-pet-project' 
```

Comment:
What if we will have a different group or cluster-name? I think it will be more useful to think about it as variables. It bring us flexebility without change the code